akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -1178,9 +1178,11 @@ private[akka] class ActorSystemImpl(
      *    when the extension cannot be found at all we throw regardless of this setting)
      */
     def loadExtensions(key: String, throwOnLoadFail: Boolean): Unit = {
+
       immutableSeq(settings.config.getStringList(key)).foreach { fqcn =>
         dynamicAccess.getObjectFor[AnyRef](fqcn).recoverWith {
-          case _ => dynamicAccess.createInstanceFor[AnyRef](fqcn, Nil)
+          case firstProblem =>
+            dynamicAccess.createInstanceFor[AnyRef](fqcn, Nil).recoverWith { case _ => Failure(firstProblem) }
```

Comment:
Added second recover so that the first exception is logged. That is probably not great for failing extensions defined in Java.